chapters/batching.rst

```diff
@@ -0,0 +1,131 @@
+=============================================
+ Understanding Batching in Jina
+=============================================
+
+Some :term:`Executors` in Jina may profit from the capacity to process multiple instances of their input at the same time.
+This can lead to a speed up and higher throughput.
+
+In order to provide this capability, the :term:`Driver` extracts all the needed information from all the :term:`Document` in a :term:`Request`. [see `guide on request size`_].
+
+Then all these instances (one per :term:`Document`) is passed to the :term:`Executor`. Then the :term:`Executor` decides how it wants to
```

Comment:
are passed

Follow that, the :term:`Executor` (sounds nice then two sentences with then)